lib/components/Provider/index.tsx

```diff
@@ -72,6 +73,7 @@ type Props = (
   initialLoadMaxAge?: Date;
   playSoundOnNewNotification?: boolean;
   newNotificationSoundPath?: string;
+  connectionMode?: 'live' | 'mocked';
```

Comment:
Please pick a better term. The mode is not about the connection (e.g. HTTP vs HTTPS), it's about the whole component and how it operates.